src/esp/sim/Simulator.cpp

```diff
@@ -145,7 +146,7 @@ void Simulator::reconfigure(const SimulatorConfiguration& cfg) {
   sceneID_.push_back(activeSceneID_);
 
   if (config_.createRenderer) {
-    if (!context_) {
+    if (!context_ && !Magnum::GL::Context::hasCurrent()) {
       context_ = gfx::WindowlessContext::create_unique(config_.gpuDeviceId);
     }
 
```

Comment:
Will we need some ifdef for that gl context code for the Python Viewer I wonder...